### PR TITLE
Feature/pattern importer

### PR DESCRIPTION
**Pull request by vibhavsalelkar**

### CODE REVIEW

File: includes/layout/functions.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+if ( ! function_exists( 'rbea_block_templates_log' ) ) :
+
+	/**
+	 * Log
+	 *
+	 * @param string $message Log message.
+	 */
+	function rbea_block_templates_log( $message = '' ) {
+		if ( rbea_block_templates_doing_wp_cli() ) {
+			WP_CLI::line( $message );
```
Comment (ratnesh-kadam) [inline includes/layout/functions.php]: @vibhavsalelkar we haven't registered any wp cli command, please remove this code.

File: includes/layout/functions.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Useful functions
+ *
+ * @since 1.0.0
```
Comment (ratnesh-kadam) [inline includes/layout/functions.php]: @vibhavsalelkar update the version number

Comment (vibhavsalelkar) [reply includes/layout/functions.php]: updated in new commit

File: includes/layout/functions.php (hunk 1 above)
Comment (vibhavsalelkar) [reply includes/layout/functions.php]: removed unwanted codes, comments and logs

**Review by Nayan-Gaikwad — APPROVED** [review]

**Pull request merged**